@navikt/core/react/src/form/file-upload/FileUpload.tsx

```diff
@@ -0,0 +1,201 @@
+import cl from "clsx";
+import React, { ChangeEvent, forwardRef, useRef, useState } from "react";
+import { BodyShort, ErrorMessage, Label } from "../../typography";
+import { useFormField } from "../useFormField";
+import { FileUploadContext } from "./context";
+import Dropzone from "./parts/Dropzone";
```

Comment:
Flyttet "sub-komponenter" i en egen mappe `parts` for å lette se hva som er "hovedkomponent"